### ISSUE TITLE
"wallet is out of sync" when using the Trezor with wrong passphrase.

### ISSUE THREAD

**Issue by Transisto** [mycelium-com/wallet#328]
"The transaction was rejected by the network. This indicate that your ."

Well no necessarily, when spending from the trezor with wrong passphrase this is the error give at last step.

Low priority, Just letting you know.

**Comment by Giszmo** [comment]
not having worked with trezor, I'm not sure if others understand this bug report. Clear way to reproduce would probably be appreciated. Otherwise thanks for reporting this.

**Comment by Rassah** [comment]
It's not so much a bug as a feature. There's no way to know if the password you entered is wrong, since any password will create a working private key, but it's not until you sign and try to send with it that you would know whether it was a valid signature, and thus a valid password. The "feature" in this is that it makes it very difficult to guess passwords, since you won't know if you guessed the right one until after you try to broadcast it.

**Comment by Transisto** [comment]
I don't mind it not working obviously, but I meant the error message should
allow for some vagueness and not 100% assume that the wallet is out of
sync.

**Comment by Giszmo** [comment]
This was a duplicate of #169 I believe? It errors out with "Probably wrong passphrase". Let us know if it needs more fixing.
